components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -12,6 +12,8 @@ import (
 	"sort"
 	"time"
 
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/auditlog"
```

Comment:
nit: can you please put this import back to among the other control-plane imports?